rust/windows-client/docs/manual_testing.md

```diff
@@ -84,4 +84,30 @@ This is a list of all the on-disk state that you need to reset to test a first-t
 
 # Tunneling
 
-TODO
+If you can't test with resources that respond to ping, curl is fine too.
+
+- The tunnel can route public-routable IPs, e.g. 1.1.1.1, for public resources
+- All resources accessed by domain will get a private network address, e.g. 10.0.14.19, even public resources
```

Comment:
I know these are internal docs, but might be good to have the realistic addresses here.

```suggestion
- All resources accessed by domain will get a CGNAT network address, e.g. 100.64.96.19, even public resources
```